webapp/lib/controller.dart

```diff
@@ -308,6 +294,32 @@ void initUI() {
       systemMessages.addAll(updatedMessages.where((m) => !m.expired));
       command(UIAction.updateSystemMessages, SystemMessagesData(systemMessages));
     });
+
+  platform.listenForUserConfigurations((userConfigurations) {
+    var defaultConfig = userConfigurations.singleWhere((c) => c.docId == 'default', orElse: () => null);
+    var userConfig = userConfigurations.singleWhere((c) => c.docId == signedInUser.userEmail, orElse: () => null);
+    if (defaultConfig == null && userConfig == null) {
+      return;
+    }
+    defaultUserConfig = defaultConfig ?? defaultUserConfig;
+    currentUserConfig = userConfig ?? currentUserConfig;
```

Comment:
What if `defaultConfig` is not `null` and `currentConfig` is `null` or vise-versa?
Are `defaultUserConfig` and `currentUserConfig` defined as non-`null` somewhere else prior to this?
I'm guessing they are and I'm just not seeing it.

Reply:
good catch, it's true I'm not initialising `defaultUserConfig` and `currentUserConfig` anywhere, and they should be, otherwise on the following line, the `currentConfig` can switch to being null, which shouldn't happen. I've fixed this now, thank you!